### PR TITLE
Change readme

### PR DESCRIPTION
**Pull request by Cheshiriks**

### CODE REVIEW

**Review by sanyavertolet — APPROVED** [review]

**Comment by sanyavertolet** [comment]
To my mind, it is better to add a reference to `save-frontend`'s readme

File: save-deploy/README.md [hunk 1]
```diff
@@ -109,8 +109,8 @@ Usually, not the whole stack is required for development. Application logic is p
    * `context`: add `/sec/**, /oauth2/**, /login/oauth2/**` to the list;
    * `target`: change to [`http://localhost:5300`](http://localhost:5300) (the
      default gateway URL); 
-   * `onProxyReq`: drop the entire callback, since all auth headers (`X-Authorization-Id`,
-     `X-Authorization-Name` and `X-Authorization-Roles`) will be set by the gateway now (the gateway
+   * `onProxyReq`: drop the entire callback, since both headers (`Authorization`
+     and `X-Authorization-Source`) will be set by the gateway now (the gateway
```
Comment (sanyavertolet) [inline save-deploy/README.md:113]: As far as I know, headers have been changed, as @nulls once said

File: save-frontend/README.md [hunk 2]
```diff
@@ -120,36 +120,4 @@ onProxyReq: (proxyReq, req, res) => {
 Thus, we add `Authorization` and `X-Authorization-Source` headers that correspond with `admin` user headers.
 
 ### Using OAuth with a local deployment (`api-gateway` on)
```
Comment (sanyavertolet) [inline save-frontend/README.md:122]: Please return it to save-frontend as it is frontend configuration

File: save-deploy/README.md (hunk 1 above)
Comment (nulls) [reply save-deploy/README.md:113]: yes, it was

Comment (nulls) [reply save-deploy/README.md:113]: @Cheshiriks can you please restore?

**Pull request merged**